PowerEditor/installer/nativeLang/german.xml

```diff
@@ -505,7 +505,7 @@ Translation note:
 				<Item id="1658" name="&amp;Unterverzeichnisse"/>
 				<Item id="1659" name="&amp;Versteckte Ordner"/>
 				<Item id="1624" name="Suchmodus"/>
-				<Item id="1625" name="Norma&amp;l"/>
+				<Item id="1625" name="&amp;Normal"/>
```

Comment:
Have you tried your own PR?
This modification has the same problem as before but now it conflicts with the `<Item id="1620" name="Suchen &amp;nach:"/>`.

You must find an unique letter for the access key inside the whole `<Find ... </Find>` section.